x-pack/plugins/lens/public/pie_visualization/render_helpers.ts

```diff
@@ -10,10 +10,10 @@ import { Datatable, DatatableColumn } from 'src/plugins/expressions/public';
 import { LensFilterEvent } from '../types';
 
 export function getSliceValue(d: Datum, metricColumn: DatatableColumn) {
-  if (typeof d[metricColumn.id] === 'number' && d[metricColumn.id] !== 0) {
+  if (typeof d[metricColumn.id] === 'number') {
     return d[metricColumn.id];
   }
-  return Number.EPSILON;
+  return 0;
```

Comment:
This is a duplicate of the above logic, including an exactly matching parameter signature. It's a preexisting issue, not the PR's doing, though to follow the DRY principle, yet maybe it's a good time to have just one function for this

Reply:
I will use the same function that we use in visualize